packages/apps/types/src/serializer/serializers/default.ts

```diff
@@ -87,7 +87,7 @@ const deserializeEchoObject = (parsed: any, newId?: boolean): Expando => {
     core.id = id;
   }
   getMeta(deserializedObject).keys.push(...(meta?.keys ?? []));
-  const typeRef = getTypeRef(type);
+  const typeRef = internalDecodeReference(type);
```

Comment:
maybe todo to remove usage? I think this was the only external usage of the api?